src/Sulu/Bundle/AudienceTargetingBundle/Tests/Functional/Controller/AdminControllerTest.php

```diff
@@ -15,6 +15,13 @@
 
 class AdminControllerTest extends SuluTestCase
 {
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->initPhpcr();
+        $this->purgeDatabase();
```

Comment:
Copied the code from somewhere... Seems like `purgeDatabase` doesn't delete anything PHPCR related, because without `initPhpcr` this test is not working. Can still turn it around to avoid confusion.

Reply:
Ok good to know would be great to turn it around to avoid confusion :)